### DISCUSSION TITLE
Question about reviewing cards and workflow

### DISCUSSION THREAD

**Discussion by Celebes** [mortii/anki-morphs#261] [Q&A] [answered]
So I have read the entire documentation and I have a question regarding [Usage -> Reviewing cards](https://mortii.github.io/anki-morphs/user_guide/usage/reviewing-cards.html). The page talks in details what to do in various worst case scenarios, and maybe I am dumb, but after reading that I am not sure how to approach T1 cards that I want to study? 

My first idea was to create 2 separate decks:
* subs2srs - deck with all the "raw" cards imported from subs2srs
* main deck - T1 cards selected to study, moved here from subs2srs and tagged with "audio-sentence-card" tag

Then I wanted to create 2 separate AnkiMorphs filters: read + write filter for cards without "audio-sentence-card" tag and read only filter for cards with "audio-sentence-card" tag. This way I could sift through the "subs2srs" decks to find T1 cards and move them to the main deck for later without studying them immediately.

But I start to think that all of this is an overkill... Should I just keep everything in a single deck ("subs2srs deck") and click again/easy/good/hard on T1 cards when I see them and that's it?

How does your workflow look like?

**Comment by mortii** [comment]
> But I start to think that all of this is an overkill... Should I just keep everything in a single deck ("subs2srs deck") and click again/easy/good/hard on T1 cards when I see them and that's it?

Yes, just use one deck. AnkiMorphs takes care of sorting the cards, so extracting them to a separate deck would indeed be a lot of unnecessary work.

> How does your workflow look like?

![Screenshot 2024-07-25 122711](https://github.com/user-attachments/assets/d4a045c2-1901-4e94-be67-fcda0c54dcb3)

I study the "My Japanese Sentences" deck everyday, which pulls cards from its six subdecks. All the subdecks have sub2srs cards with the same note type, so I only have one note filter.

Using sub-decks does make things a little more complicated because you have to [adjust one deck option](https://mortii.github.io/anki-morphs/user_guide/setup/decks.html#deck-options), so just merging all your cards into one simple deck would be easier.

Does that make sense? I'm pretty bad at guides, so any suggestions for improving it would be very welcome :)

**Reply by Celebes** [reply]
Okay so I just treat T1 cards as regular study cards and pick again/easy/good/hard to move them from "new" to learning? Can "not-new" cards still be modified by AnkiMorphs? For example let's say I encounter a good T1 card, I start to study it and it is no longer new - can `am-unknowns` field still be edited automatically by AnkiMorphs and I will lose info about target word once the card becomes T0 in the future?

**Reply by mortii** [reply, marked as answer]
> Okay so I just treat T1 cards as regular study cards and pick again/easy/good/hard to move them from "new" to learning?

Yep, just study them like any other anki card, and recalc when you are done with your study session.

>  For example let's say I encounter a good T1 card, I start to study it and it is no longer new - can `am-unknowns` field still be edited automatically by AnkiMorphs and I will lose info about target word once the card becomes T0 in the future?

Great question. Right now `am-unknowns` stays the same after you have studied the card, i.e., no information is lost. However, after a lot of discussion about this in #221, it will be modified slightly in [v3](https://github.com/mortii/anki-morphs/issues/222). Basically it will be replaced with two new extra fields:
1. `am-study-morphs`, which will have the same properties as the current  `am-unknowns`
2.  `am-unknown-morphs` which will always update, even after having studied the card for the first time

Sorry about the info dump, it's probably very confusing :/

**Reply by Celebes** [reply]
No, I think it actually cleared my doubts, so thanks. The idea to split the field into two is good because then people will be able to see which target word the card contained even after it becomes T0 for them.

**Reply by mortii** [reply]
Great! Any suggestions for changes to the guide that would have prevented any of these doubts/confusions?

**Reply by Celebes** [reply]
Yeah, just adding a section like "Encountering T1 Cards You Do Understand" that sums up what we've talked about here would be helpful. Because as I said the guide now doesn't describe "the happy path" and only talks about how to handle "bad" scenarios.

So let's say I am just starting out, I've encountered only very easy cards so far and I skipped them with "mark as known and skip" option. Then suddenly a T1 card appears. Let's say I don't know word "pancakes" and I can see "I love pancakes" sentence. I wasn't able to understand it from context, but I think it is a card worthy of studying. What do I do? Do I add some more info to the card's back side like "pancakes" dictionary definition and translation to my native language and then press "Repeat" so it becomes a not-new card? etc.

**Reply by mortii** [reply]
Makes sense, I'll add something like that :+1: 

Thanks!

**Reply by mortii** [reply]
> and then press "Repeat" so it becomes a not-new card?

just to answer this quickly: answering the card in any way will move it from `new` -> `review`, so it is pure preference (I usually press "good")

**Reply by mortii** [reply]
@Celebes is this okay?

> ### Encountering Suitable 1T Cards
>
>If you come across a new card with only one unknown and it seems reasonable, treat it like any other new Anki card and
answer it accordingly. For more information on handling new cards, refer to the [Anki studying guide](https://docs.ankiweb.net/studying.html#learningrelearning-cards).

**Reply by mortii** [reply]
I'll just add it as is and you can give any corrections later if needed.
